tmf/org.eclipse.tracecompass.tmf.ui/src/org/eclipse/tracecompass/tmf/ui/widgets/timegraph/widgets/TimeGraphControl.java

```diff
@@ -2333,8 +2334,17 @@ protected void drawMarker(IMarkerEvent marker, Rectangle bounds, ITimeDataProvid
             case SymbolType.CIRCLE:
                 SymbolHelper.drawCircle(gc, color, symbolSize, rect.x, y);
                 break;
-            default:
+            case SymbolType.DIAMOND: // Fall-through
+            case SymbolType.NONE:
```

Comment:
NONE cannot really happen here because it is checked in the containing if-clause.
If we must put it explicitly (perhaps for static code quality checkers?), it would be clearer to separate it with just a break to make it clear that it does not draw anything.

In fact, for NONE it will draw the name of the annotation (from the annotation model, not the style)...

Reply:
oh I completely missed that. No, it doesn't complain about a missing `case SymbolType.NONE`, so I just removed it.